metrics/exp/exp.go

```diff
@@ -108,6 +122,10 @@ func (exp *exp) publishGaugeFloat64(name string, metric metrics.GaugeFloat64) {
 	exp.getFloat(name).Set(metric.Value())
 }
 
+func (exp *exp) publishGaugeInfo(name string, metric metrics.GaugeInfo) {
+	exp.getInfo(name).Set(fmt.Sprintf("%s", metric.Value()))
```

Comment:
The value is already a string, so this could be simplified to `exp.getInfo(name).Set(metric.Value())`